bulkwebhook/bulk_webhook/doctype/kafka_settings/kafka_settings.py

```diff
@@ -1,9 +1,12 @@
 # Copyright (c) 2022, Aakvatech and contributors
 # For license information, please see license.txt
 
-# import frappe
+import frappe
+import bulkwebhook
 from frappe.model.document import Document
 
 
 class KafkaSettings(Document):
-    pass
+    def on_update(self):
```

Comment:
`on_trash` as well? or we can just extend the `clear_cache` method.

Reply:
Would've liked to use https://github.com/frappe/frappe/pull/17012 instead. But think I'll just add on_trash too for simplicity